manifests/dev/vsphere-7.0u2/guestcluster/1.17/pvcsi.yaml

```diff
@@ -238,8 +238,7 @@ spec:
             - "--v=4"
             - "--timeout=300s"
             - "--csi-address=$(ADDRESS)"
-            - "--enable-leader-election"
-            - "--leader-election-type=leases"
+            - "--leader-election"
```

Comment:
`leader election requires` following RBAC rule on the service account? Can you add this rule? 

```
  - apiGroups: ["coordination.k8s.io"]
    resources: ["leases"]
    verbs: ["get", "watch", "list", "delete", "update", "create"]
```

Refer https://github.com/kubernetes-csi/external-provisioner/blob/master/deploy/kubernetes/rbac.yaml#L91-L93

Reply:
@divyenpatel `leader-election-type` parameter was removed in `v2.1.0`. I guess this RBAC is old - https://github.com/kubernetes-csi/external-provisioner/blob/v2.1.0/cmd/csi-provisioner/csi-provisioner.go#L72

Reply:
so RBAC rules regarding `coordination.k8s.io` are no longer required?